docs/src/examples/examples.md

```diff
@@ -0,0 +1,54 @@
+## Introduction
+
+```@contents
+Pages = ["examples.md"]
+Depth = 3
+```
+
+The folder `/examples` contains a collection of model files that are available from different
+sources:
+
+- [SpaceEx webpage](http://spaceex.imag.fr/download-6), small selection of
+  examples that is part of the VM Server distribution and can be executed directly
+  from the SpaceEx web interface.
+
+### Remarks
+
+The examples consist of single hybrid automata that are constructed via flattening (parallel composition). This can be done via SpaceEx executable file with the command
+
+`spaceex -g name.cfg -m name.xml --output-system-file new_name.xml`
+
+However, note that the flattening process change the original model and may induce parsing errors. The parsing errors only appear when the constructed model is visualized/analyzed with the Model Editor or/and the Web Interface. There are no parsing errors with the source code/executable SpaceEx. A list of identified parsing problems follows below.
+
+1. Special symbols, e.g. ~, _ in the variable and location names
+2. Characters and symbols in the notes
```

Comment:
What is the difference between characters and symbols?

Reply:
Probably the best thing is to say no special languages and use latin alphabet.

To answer your question, from my standpoint, symbols correspond to mathematical operations (,},/,*,+,_ and characters to non-latin characters, e.g. Greek or Russian letters